test/corpus/pluscal/assert.txt

```diff
@@ -0,0 +1,25 @@
+==================|||
+PlusCal Assert
+==================|||
+---- MODULE Test ----
+(* --algorithm Test
+begin
+  assert TRUE
+end algorithm *)
+====
+
+--------------------------------------------------------------------------------
```

Comment:
Need to add the suffix on these separators as well

Reply:
done